build/templates/functions.rst/lock_session_wrapper.rst.mako

```diff
@@ -0,0 +1,64 @@
+.. py:method:: lock_session(caller_has_lock=None)
+
+    | Obtains a multithread lock on the device session. Before doing so, the
+      software waits until all other execution threads release their locks
+      on the device session.
+    | Other threads may have obtained a lock on this session for the
+      following reasons:
+
+    -  The application called the :py:meth:`nidcpower.Session.lock_session` method.
+    -  A call to NI-DCPower locked the session.
+    -  A call to the IVI engine locked the session.
```

Comment:
This line doesn't make sense in the context of Python APIs. If it was extracted, you can drop.

Reply:
It came from the extracted metadata, but copied here. How does it not make sense? How should it be changed?

Reply:
>   -  A call to the IVI engine locked the session.
 
How can there be a call to the IVI engine from Python?